maint_tools/test_docstrings.py

```diff
@@ -231,7 +231,7 @@
     "sklearn.utils.validation.check_memory",
     "sklearn.utils.validation.check_random_state",
     "sklearn.utils.validation.column_or_1d",
-    "sklearn.utils.validation.has_fit_parameter",
+    # "sklearn.utils.validation.has_fit_parameter",
```

Comment:
@reshamas I guess what @ogrisel is asking, is to revert the changes which are not related to the function you're directly addressing in this PR, and to open a separate PR for them, which I agree with, especially if it helps with fewer merge conflicts with other PRs.

Could you elaborate what the output you see is, so that we could maybe help you find the relevant places to fix?